instrumentor/controllers/instrumentationdevice/common.go

```diff
@@ -210,6 +210,12 @@ func reconcileSingleWorkload(ctx context.Context, kubeClient client.Client, runt
 		return err
 	}
 
+	if runtimeDetails.Spec.RuntimeDetails[0].Language == "nginx" {
```

Comment:
I think I may need to run all the runtimeDetails.Spec.RuntimeDetails, even though I'm not sure when it's more then one.